### ISSUE TITLE
line 13: 36766 Killed: 9

### ISSUE THREAD

**Issue by jonalange** [imapsync/imapsync#456]
I'm removing duplicates in a pretty big mailbox. Somebody duplicated a folder with 10k emails, so now there are two folders with 10k each. I currently resolve the duplicates like you mentioned here: https://github.com/imapsync/imapsync/issues/213#issuecomment-2112263900

Every 10 minutes the whole process stops for no obvious reason.
Last line that's shown (before that are just "message copied to ...")  `mail_duplicate_remover: line 13: 36766 Killed: 9` followed by my whole script including my credentials etc.

Line 13 in my case is the last line and just says `--delete1`

Whole code (except logins):
```
./imapsync  --host1 domain.de --user1 user@email.de --password1 'abc' --ssl1 --port1 993 --sep1 '/' \
            --host2 domain.de --user2 user@email.de --password2 'abc' --ssl2 --port2 993 --sep2 '/' \
            --nofoldersizes \
            --nofoldersizesatend \
            --folder 'INBOX/04_Mitarbeiter/Jona/Gesendete Elemente Duplikate' \
            --f1f2 'INBOX/04_Mitarbeiter/Jona/Gesendete Elemente Duplikate'='Gesendete Elemente' \
            --delete1
```

Running it on macOS in terminal

**Comment by gilleslamiral** [comment]
It looks like a memory issue on Mac, described at
https://github.com/imapsync/imapsync/issues/312
Can you try with the imapsync binary? https://imapsync.lamiral.info/dist/
Or on another system?

**Comment by jonalange** [comment]
> It looks like a memory issue on Mac

You are right. When I run the script on a linux based Synology NAS it does work without a single interruption.
